webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -647,7 +648,90 @@ private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, Scram
 
             offsetTop += fontSize + marginBottom;
         }
+        
+        // the moves specs inside the table are not exactly with the rules.
+        // they are rendered separately, then rules are added above it
+        
+        // table
+        int tableHeight = 160;
+        final int tableLines = 8; // currently, we have 8 lines (3 face moves, 3 rotations and titles)
+        
+        Font smallfont = new Font(FontFamily.HELVETICA, 10);
+        PdfPTable table = new PdfPTable(8);
+        table.setTotalWidth(new float[]{25, 25, 25, 25, 25, 25, 25, competitorInfoLeft-left-7*25}); // the last part makes the moves specs fit the rules rectangle
+        table.setLockedWidth(true);
 
+        String faceMoves[] = {
```

Comment:
nit: `String ...[] -> String[] ...`

Reply:
Fixed. I'd like to know where I can find this standards so I don't struggle with Travis anymore :)